packages/utils/src/route.js

```diff
@@ -47,7 +47,7 @@ function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-')
         paths.shift()
       } // clean first / for parents
       routesIndex.forEach((r) => {
-        const i = r.indexOf('index') - start //  children names
+        const i = r.indexOf('inxdex') - start //  children names
```

Comment:
typo?

Reply:
oh lol, yeah :see_no_evil: 
```suggestion
        const i = r.indexOf('index') - start //  children names
```